src/postman/operation.ts

```diff
@@ -2,24 +2,40 @@ import {
   HttpParamStyles,
   HttpSecurityScheme,
   IHttpHeaderParam,
+  IHttpOperation,
   IHttpOperationRequestBody,
   IHttpParam,
   IHttpPathParam,
   IHttpQueryParam,
   IMediaTypeContent,
 } from '@stoplight/types';
-import { Collection, Item, ItemGroup, RequestAuth, Url } from 'postman-collection';
+import { Collection, CollectionDefinition, Item, ItemGroup, RequestAuth, Url } from 'postman-collection';
 import { transformRequest } from './transformers/request';
 import { transformResponse } from './transformers/response';
 import {
   isPostmanSecuritySchemeEqual,
+  PostmanSecurityScheme,
   transformSecurityScheme,
   transformSecuritySchemes,
 } from './transformers/securityScheme';
 import { transformServer } from './transformers/server';
 import { PostmanCollectionHttpOperationTransformer } from './types';
 import { transformDescriptionDefinition, traverseItemsAndGroups } from './util';
 
+export const transformPostmanCollectionOperations = (document: CollectionDefinition): IHttpOperation[] => {
+  const collection = new Collection(document);
+  const resolvedCollection = new Collection(collection.toObjectResolved({ variables: collection.variables }, []));
+
+  const securitySchemes = transformSecuritySchemes(collection);
+  const operations: IHttpOperation[] = [];
+
+  traverseItemsAndGroups((resolvedCollection as unknown) as ItemGroup<Item>, item =>
+    operations.push(transformItem(item, securitySchemes)),
+  );
+
+  return operations;
+};
+
 export const transformPostmanCollectionOperation: PostmanCollectionHttpOperationTransformer = ({
```

Comment:
Do we really need a function transforming a single item? I really do not see this happening and this is not even used in `transformPostmanCollectionOperations` — I guess we can probably remove this function.

Reply:
That function is part of the `http-spec`'s core interface. Basically `http-spec` exposes two set of functions:
1. `transform%deathmetal%Operation`
2. `transform%deathmetal%Operation`

Where `%deathmetal%` is one of `Oas2`, `Oas3`, `PostmanCollection`.

https://github.com/search?q=org%3Astoplightio+transformOas2Operation&type=Code
Even in `prism` we currently* use single operation transformation.

* I refactored that here https://github.com/stoplightio/prism/compare/feat/postman-collection to use the bulk transformation function

Reply:
> That function is part of the http-spec's core interface


Ah ok, meh, what an usesless function anyway.